code/ui/InfoFeed.cs

```diff
@@ -23,15 +24,19 @@ public virtual Panel AddEntry(Client leftClient, Client rightClient, string meth
             bool isLeftLocal = leftClient == Local.Client;
             bool isRightLocal = rightClient == Local.Client;
 
+            TTTPlayer leftPlayer = leftClient.Pawn as TTTPlayer;
+
             Label leftLabel = e.AddLabel(isLeftLocal ? "You" : leftClient.Name, "left");
-            leftLabel.Style.FontColor = (leftClient.Pawn as TTTPlayer).Role.Color;
             leftLabel.SetClass("me", isLeftLocal);
+            leftLabel.Style.FontColor = leftPlayer.Role is NoneRole ? Color.White : leftPlayer.Role.Color;
```

Comment:
`NoneRole` has a transparent color in the class, but we use `Color.White` here, any reason why we can't just change it to be `Color.White`? I'm assuming it's probably because it's different colouring we want to use on leaderboard vs. infofeed.

Would it be worth to have to separate colours on each class then?

Just some thoughts.